kobo/apps/accounts/templatetags/customize_email_content.py

```diff
@@ -37,3 +37,9 @@ def convert_placeholders(section_content, activate_url, user):
         '##activate_url##', activate_url
     ).replace('##user##', user.username).replace("&#x27;", "\'")
 
+
+@register.simple_tag(takes_context=True)
+def ip_address(context):
+    request = context['request']
+    address = request.META.get('REMOTE_ADDR')
```

Comment:
Please have a look at [this utility](https://github.com/kobotoolbox/kobocat/blob/0d95c31961ed13f7ae91b2154c8dbbd99b4a5d3a/onadata/libs/utils/viewer_tools.py#L90) in kobocat (comes from SO 😄 ).

IIRC, The ip address can be different if python app is behind a proxy.